src/plugins/console/api_server/es_5_0/cat.js

```diff
@@ -0,0 +1,56 @@
+let _ = require("lodash");
+
+function addSimpleCat(endpoint, api, params, patterns) {
+  var url_params = {"help": "__flag__", "v": "__flag__", "bytes": ["b"]};
+  _.each(params || [], function (p) {
+    if (_.isString(p)) {
+      url_params[p] = "__flag__";
+    }
+    else {
+      var k = Object.keys(p)[0];
+      url_params[k] = p[k];
+    }
+  });
+  api.addEndpointDescription(endpoint, {
+    match: endpoint,
```

Comment:
It looks like this is so because `methods` defaults to `['GET']` as per https://github.com/ycombinator/kibana/blob/gh-6913/src/plugins/console/api_server/api.js#L27-L31. That being said, I agree that this is inconsistent — sometimes `methods: ['GET']` is explicitly specified in the call to `addEndpointDescription` and sometimes it is not.

I think this should be cleaned up and made consistent but I'd like to scope this cleanup to the code under the `es_5_0` folder only, as the scope of this PR is to make Console work with ES 5.0 APIs. What do you think?

I think its a good idea to make this consistent and explicitly specify the allowed methods. So I'll update the 5.0 codebase to do that (but I'm not going .